javascript/src/main/components/OfficeHours/OfficeHourTable.js

```diff
@@ -37,14 +37,14 @@ export default ({officeHours,admin,deleteOfficeHour}) => {
 
     if (admin) {
         columns.push({
-            text: "Delete",
+            text:  "Delete",
             isDummyField: true,
             dataField: "delete",
             formatter: (_cell, row) => renderDeleteButton(row.id)
         });
     }
 
     return (
-        <BootstrapTable keyField='id' data={officeHours} columns={columns} />
+        <BootstrapTable keyField='id' data={officeHours} columns={columns} striped={true}/>
```

Comment:
The `={true}` is redundant in React. If you change this to `striped`, React will implicitly treat this as `striped={true}`.
```suggestion
        <BootstrapTable keyField='id' data={officeHours} columns={columns} striped />
```